### PR TITLE
865 incompatible pyarrow version causing issues with parquet file reading in fink science library

### LINKED ISSUE #865: Incompatible PyArrow version causing issues with Parquet file reading in fink_science library
**Describe the issue**
_When trying to read a Parquet file using the `fink_science` library, I'm encountering an issue where the library is unable to find a usable engine for the Parquet file. The error message indicates that a suitable version of PyArrow or Fastparquet is required, and that the currently installed version of PyArrow (9.0.0) is not compatible with the required version (10.0.1)._

**Expected behaviour**
_I expected the `fink_science` library to be able to read the Parquet file without any issues, as I have PyArrow installed in my environment._

**Actual behaviour**
_The actual behaviour is that the `fink_science` library is unable to find a usable engine for the Parquet file, and is raising an `ImportError` with the following message:_

```
ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
A suitable version of pyarrow or fastparquet is required for parquet support.
Trying to import the above resulted in these errors:
 - Pandas requires version '10.0.1' or newer of 'pyarrow' (version '9.0.0' currently installed).
```

**To Reproduce**

Launch the CI script with commit `117d5f80b41decc35072e5e95c5238f5f55d24a7` of fink-broker main (master) branch

```
3. The code failed with the `ImportError` mentioned above.

**System Information:**
 - Operating System: [`Linux`]
 - Fink Version: [`0.1.0`]
 - Occurred on which branch and with what commit: [`master-abc456d`]

**Additional context**
_The issue seems to be related to the version of PyArrow installed in my environment. I need to upgrade PyArrow to version 10.0.1 or newer to resolve the compatibility issue with the `fink_science` library._

### PR DESCRIPTION
**Pull request by fjammes**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): Closes #865 

## How was this patch tested?

CI for science was launche manually with success.

### CODE REVIEW

**Comment by JulienPeloton** [comment]
@fjammes -- the CI is failing because of a missing file:

```
Context "kind-kind" modified.
/home/runner/work/_temp/11f0b535-85ed-4074-9992-09ff61e98e76.sh: line 3: /argo-submit.sh: No such file or directory
```

Can you fix this?

**Comment by JulienPeloton** [comment]
@fjammes - the CI is failing because it cannot find the image:

```
  Warning  Failed       55m (x4 over 56m)  kubelet            Failed to pull image "gitlab-registry.in2p3.fr/astrolabsoftware/fink/fink-
broker-noscience:v3.1.2-rc1-57-g60a6251": rpc error: code = NotFound desc = failed to pull and unpack image "gitlab-
registry.in2p3.fr/astrolabsoftware/fink/fink-broker-noscience:v3.1.2-rc1-57-g60a6251": failed to resolve reference "gitlab-
registry.in2p3.fr/astrolabsoftware/fink/fink-broker-noscience:v3.1.2-rc1-57-g60a6251": gitlab-registry.in2p3.fr/astrolabsoftware/fink/
fink-broker-noscience:v3.1.2-rc1-57-g60a6251: not found
```
Problem of reference?

File: e2e/argocd.sh [hunk 1]
```diff
@@ -47,7 +50,7 @@ argocd app sync fink
 argocd app sync strimzi minio-operator spark-operator
 
 # TODO Try to make it simpler, try a sync-wave on Strimzi Application?
-# see https://github.com/argoproj/argo-cd/discussions/16729
+# see https://github.com/argoproj/argo-cd/discussions/16729argocd app set
```
Comment (JulienPeloton) [inline e2e/argocd.sh:53]: Malformed link.

Comment (fjammes) [reply e2e/argocd.sh:53]: :+1:

**Pull request merged**